### PR TITLE
Temperate world away mission 

### PR DESCRIPTION
**Pull request by CaraS212**
Adds a new joinable temperate planet away. 100^2 x 3 Z levels with new animals, mobs, loot, and a couple easter eggs. 
Stairs included for pulling out injured individuals.
Adds non snowy pine tree sprites
adds comments to planet turf given by glloyd

many sprites grabbed from Civ13 and Eris.

Remaining Issue:
Shuttles not able to land on normal shuttle_landmark and uses the automatic version as a temporary solution

### CODE REVIEW

File: maps/away/forest/forest.dm [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	base_area = /area/planet/forest
+	base_turf = /turf/simulated/floor/planet/dirt/temperate
+
+/datum/map_template/ruin/away_site/caraway_forest
+	name = "Temperate Planet"
+	id = "awaysite_caraway_forest"
+	description = "3z level planet with cave, forest surface, and 2 story buildings."
+	suffixes = list("forest/forest1.dmm", "forest/forest2.dmm", "forest/forest3.dmm")
+	accessibility_weight = 11
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```
Comment (CaraS212) [inline maps/away/forest/forest.dm:97]: remove when shuttles land on the nav points

**Comment by Glloyd** [comment]
You have conflicts that will need to be resolved before this PR can go further. The icon conflicts need to be resolved manually, best way is to save your changes separately, revert your changes to those files, update, and then manually paste in your new sprites into those .dmis. It sucks, and is why we avoid editing Bay icon files wherever possible.

**Comment by CaraS212** [comment]
I'll look at the icons in the morning but what is the conflict with uristanimals.dm? The only change there is a new mob.

**Comment by Glloyd** [comment]
Someone else has edited those same lines, which is causing a conflict. Your additions to that file should probably be in humanhostiles.dm anyways though, for the sake of organization.

**Comment by CaraS212** [comment]
Moved the pirates in simple animals to humanhostiles, no idea why that is coming up as a conflict.
grabbed the latest DMIs that were causing error and those are now updated

**Comment by Glloyd** [comment]
finally getting back around to this, sorry for the delay

File: code/modules/urist/items/clothes/head.dm [hunk 2]
```diff
@@ -464,3 +464,251 @@ to avoid worrying about the sprites -Glloyd*/
...
+	desc = "a beautiful golden crown. This one has a diamond set in it"
+	icon_state = "gold_crown_diamond"
+	item_state = "gold_crown_diamond"
+
+/obj/item/clothing/head/urist/historic/crown/pharoah
+	name = "pharoah headdress"
+	desc = "an ancient gilded cloth headdress for a living god."
+	icon_state = "pharoah_headdress"
+	item_state = "pharoah_headdress"
+	flags_inv = HIDEEARS|BLOCKHEADHAIR
```
Comment (scrdest) [inline code/modules/urist/items/clothes/head.dm:594]: 'pharaoh'

**Pull request closed without merging**